src/wrappers/PantosSWrapper.sol

```diff
@@ -5,13 +5,13 @@ pragma solidity 0.8.26;
 import {PantosCoinWrapper} from "../PantosCoinWrapper.sol";
 
 /**
- * @title Pantos-compatible token contract that wraps the Fantom
- * blockchain network's Fantom coin
+ * @title Pantos-compatible token contract that wraps the Sonic
+ * blockchain network's S coin
  */
-contract PantosFantomWrapper is PantosCoinWrapper {
-    string private constant _NAME = "Fantom (Pantos)";
+contract PantosSWrapper is PantosCoinWrapper {
```

Comment:
` PantosSWrapper` looks generic, why not use  `PantosSonicWrapper`

Reply:
Because the name of the wrapped token is "S" and not "Sonic".
"Sonic" is only the network name.